content/faq/_index.md

```diff
@@ -60,7 +60,7 @@ If you are using a supported application framework, follow its instructions for
 
 OpenCensus provides a stand-alone application that uses a gRPC channel to communicate with the OpenCensus code linked into your application. The application displays configuration parameters and trace information in real time held in the OpenCensus library.
 
-You can learn more by visiting [zPages](/core-concepts/z-pages/).
```

Comment:
It works though, just by alias and shouldn't 404 but sure thanks for this update.

Reply:
Nvm it is tripping out apparently.

Reply:
There's one redundant `-`, removed.